tests/components/samsungtv/test_config_flow.py

```diff
@@ -71,9 +71,24 @@ def remote_fixture():
         yield remote
 
 
-async def test_user(hass, remote):
-    """Test starting a flow by user."""
+@pytest.fixture(name="remotews")
+def remotews_fixture():
+    """Patch the samsungtvws SamsungTVWS."""
+    with patch("samsungtvws.SamsungTVWS") as remotews_class, patch(
+        "homeassistant.components.samsungtv.config_flow.socket"
+    ) as socket_class:
+        remotews = mock.Mock()
+        remotews.__enter__ = mock.Mock()
+        remotews.__exit__ = mock.Mock()
+        remotews.open = mock.Mock()
```

Comment:
Attributes on the mock will return mocks by default. We can probably simplify here.

Reply:
Can remove only `.open` the other two must remain, otherwise I get `AttributeError`.